platform/security/pdp/security-pdp-xacmlrealm/src/main/java/ddf/security/pdp/xacml/processor/BalanaPdp.java

```diff
@@ -104,13 +110,20 @@ public BalanaPdp(String relativeXacmlPoliciesDirectoryPath) throws PdpException
     }
 
     /**
+     * Disable the default constructor
+     */
+    private BalanaPdp() {
```

Comment:
Why is this not needed?  It disables the default constructor which should not be used.